server/src/core/server/models/dsaReport/report.ts

```diff
@@ -0,0 +1,100 @@
+import { v4 as uuid } from "uuid";
+
+import { Sub } from "coral-common/common/lib/types";
+import { MongoContext } from "coral-server/data/context";
+import { FilterQuery } from "coral-server/models/helpers";
+import { TenantResource } from "coral-server/models/tenant";
+
+import { GQLDSAReportStatus } from "coral-server/graph/schema/__generated__/types";
+
+export interface DSAReport extends TenantResource {
+  readonly id: string;
+
+  userID: string;
+
+  createdAt: Date;
+
+  lawBrokenDescription: string;
+
+  additionalInformation: string;
+
+  commentID: string;
+
+  submissionID?: string;
```

Comment:
Why is this optional if we are always supplying one?

Reply:
the idea is that you can optionally supply the `submissionID` here in the cases when there are multiple comments reported together and you want comments to all be considered one submission. If it's not supplied, such as when it's just a single comment being reported, an id is generated down on line 54 of this file that is used, so every `DSAReport` will have a `submissionID`.

Reply:
I agree with @marcushaddon here, if every report will have a submissionID, and since this is the domain/data model (not a GraphQL/API input type) I feel it should be required.

Reply:
Thank you for reminding me...meant to follow up here, but I plan to add this as not optional in my other branch, I just merged this in before updating here. And the submissionID generation is already included!